scenes/UI/UI.gd

```diff
@@ -9,6 +9,8 @@ extends CanvasLayer
 # Called when the node enters the scene tree for the first time.
 func _ready():
 	GameState.player.connect("hull_health_changed",self, "update_health")
+	update_health(GameState.player.hull_health)
```

Comment:
Before the player health always started at 100 so we didn't need to explicitly set it. Now that the player health can be any value we need to initialize it to the correct value otherwise the GameState can have the health at one value but the UI shows it at 100.